appeals/web/src/server/appeals/appeal-details/appeal-details.mapper.js

```diff
@@ -368,6 +370,36 @@ export async function appealDetailsPage(appealDetails, currentRoute, session) {
 				});
 			}
 		}
+	} else if (
+		appealDetails.appealStatus === APPEAL_CASE_STATUS.CLOSED &&
+		appealDetails.resubmitTypeId &&
+		appealDetails.appealTimetable?.caseResubmissionDueDate
+	) {
+		const appealTypesFromId = await getAppealTypesFromId(request.apiClient, appealDetails.appealId);
+		const appealTypeById = appealTypesFromId?.find(
+			(appealType) => appealType.id === appealDetails.resubmitTypeId
+		);
+		const appealTypeText =
+			appealTypeById?.key && appealTypeById?.type
+				? `${appealTypeById.type} (${appealTypeById.key})`
+				: '';
+		const caseResubmissionDueDate = new Date(appealDetails.appealTimetable.caseResubmissionDueDate);
+
+		if (appealTypeText) {
+			statusTagsComponentGroup.push({
+				type: 'inset-text',
+				parameters: {
+					html: `<p>This appeal needed to change to ${appealTypeText}</p><p>The appellant has until ${caseResubmissionDueDate.toLocaleDateString(
+						'en-gb',
+						{
+							day: 'numeric',
+							month: 'long',
+							year: 'numeric'
+						}
+					)} to resubmit.</p>`
+				}
+			});
+		}
 	}
 
 	if (appealDetails.appealStatus === 'transferred') {
```

Comment:
Probably not for this PR, but appeal statuses should now be fetched by `APPEAL_CASE_STATUS.TRANSFERRED` in `import { APPEAL_CASE_STATUS } from 'pins-data-model';`

Reply:
yeah we need a seaprate tech debt ticket to go over status strings across the whole platform